client/state/data-layer/wpcom-http/utils.js

```diff
@@ -89,6 +89,29 @@ export const makeParser = ( schema, schemaOptions = {}, transformer = identity )
 
 		validate = data => {
 			if ( ! validator( data ) ) {
+				if ( 'development' === process.env.NODE_ENV ) {
+					// eslint-disable-next-line no-console
+					console.warn( 'JSON Validation Failure' );
+
+					validator.errors.forEach( error =>
+						// eslint-disable-next-line no-console
+						console.warn( {
+							field: error.field,
+							message: error.message,
+							value: error.value,
+							actualType: error.type,
+							expectedType: get( schema, error.schemaPath ),
+						} )
+					);
+
+					if ( undefined !== window ) {
+						// eslint-disable-next-line no-console
+						console.log( 'updated `lastValidator` and `lastValidated` in console' );
```

Comment:
This is a pretty cool idea. What about explaining what those exposed variables are good for?

> _something something_ try tweaking `lastValidator( lastValidated )` in the console and inspecting `lastValidator.errors` _something something_

Reply:
was trying to figure out where to do it. I don't want to pollute the console. docs? which docs? I was hoping this could pique someone's curiosity enough…

Reply:
An idea might be do add a README section for how to debug this and link to it from the console message. I've seen other projects do this.